Makefile

```diff
@@ -1,24 +1,26 @@
 SHORT_NAME := dockerbuilder
-DEIS_REGISTY ?= quay.io/
+DEIS_REGISTRY ?= quay.io/
 IMAGE_PREFIX ?= deis
 
 include versioning.mk
 
+#DEV_ENV_IMAGE := quay.io/deis/python-dev:v0.22.0
+DEV_ENV_IMAGE := quay.io/mboersma/python-dev:latest
```

Comment:
I'd like to replace this with an immutable image tag, once deis/docker-python-dev#11 is merged.